core/txpool/legacypool/list.go

```diff
@@ -308,11 +307,12 @@ func (l *list) Contains(nonce uint64) bool {
 //
 // If the new transaction is accepted into the list, the lists' cost and gas
 // thresholds are also potentially updated.
-func (l *list) Add(tx *types.Transaction, priceBump uint64, l1CostFn txpool.L1CostFunc) (bool, *types.Transaction) {
+func (l *list) Add(tx *types.Transaction, priceBump uint64, l1CostFn txpool.L1CostFunc, fct txpool.FeeCurrencyTranslator) (bool, *types.Transaction) {
 	// If there's an older better transaction, abort
 	old := l.txs.Get(tx.Nonce())
 	if old != nil {
-		if old.GasFeeCapCmp(tx) >= 0 || old.GasTipCapCmp(tx) >= 0 {
+		// Short circuit for majority of problems
```

Comment:
What are the "majority of problems"?

Reply:
Haha, I must have been a bit asleep when I wrote that. I'll change it to 'short circuit when it's clear that the new tx is worse'